frontend/src/components/networkpolicy/Details.tsx

```diff
@@ -129,7 +129,7 @@ export function NetworkPolicyDetails() {
               rows={[
                 {
                   name: t('Ports'),
-                  value: item.ports.map((port: NetworkPolicyPort) => (
+                  value: item.ports?.map((port: NetworkPolicyPort) => (
```

Comment:
I think we will need to add this to https://github.com/headlamp-k8s/headlamp/pull/1171/commits/e011e0a6325a4b38b355c9c4245f4a8585ba2b95#diff-8c223f7536407d20b65eb265104f27179319f1d8715cc3b31c82364a5b32347dR64 

https://github.com/headlamp-k8s/headlamp/pull/1171/commits/e011e0a6325a4b38b355c9c4245f4a8585ba2b95#diff-8c223f7536407d20b65eb265104f27179319f1d8715cc3b31c82364a5b32347dR76


https://github.com/headlamp-k8s/headlamp/pull/1171/commits/e011e0a6325a4b38b355c9c4245f4a8585ba2b95#diff-8c223f7536407d20b65eb265104f27179319f1d8715cc3b31c82364a5b32347dR92

as well.

Reply:
I have updated it. Can you test?